plans/all.fmf

```diff
@@ -0,0 +1,26 @@
+adjust:
+    when: initiator is not defined or initiator != packit
+    discover+:
+        dist-git-source: true
+        dist-git-extract: container-selinux-*/
+
+discover:
+    how: fmf
+execute:
+    how: tmt
+
+/upstream:
+    summary: Run SELinux specific Podman e2e tests on upstream PRs
+    discover+:
+        filter: tag:upstream
+    adjust:
+        enabled: false
+        when: initiator is not defined or initiator != packit
```

Comment:
```suggestion
    adjust+:
      - enabled: false
        when: initiator is not defined or initiator != packit
```
Oh this is a fun `fmf` aspect. When you had it here, `adjust` was overwriting one-another, so the `dist-git-source` one, had no effect`. Seems like you need to use the `adjust+:` here and make both of the `adjust` parts into lists

Reply:
Still need converting the other `adjust` parts ;)

Reply:
Oh lol, I guess you've found another solution. The other one would have been yo make the yaml value of `adjust` into a list. Then it's like python's `list += list` operation